### PR TITLE
Translation for home.yaml

### PR DESCRIPTION
**Pull request by Tar-Minyatur**
German translation for home.yaml

### CODE REVIEW

File: Localization/de-DE/home.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+#"Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use.#
+#
+#Available for:
+#
+#Linux
+#
+#Windows
+#
+#Mac",
+"Suchen Sie sich die Version der Software aus, die ihnen am liebsten ist und legen Sie los. Folding@home herunterzuladen ist komplett kostenlos, einfach zu installieren und die Nutzung ist sicher.
```
Comment (michael-raps) [inline Localization/de-DE/home.yaml:19]:
```suggestion
"Suchen Sie sich die Version der Software aus, die ihnen am liebsten ist und legen Sie los. Folding@home herunterzuladen ist komplett kostenlos, es ist einfach zu installieren und die Nutzung ist sicher.
```

File: Localization/de-DE/home.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Verfügbar für:
+
+Linux
+
+Windows
+
+Mac",
```
Comment (max-dau) [inline Localization/de-DE/home.yaml:27]:
```suggestion
MacOS",
```

Comment (max-dau) [reply Localization/de-DE/home.yaml:27]: technically the correct way when talking about operating systems. not in the original text though. just an option.

Comment (Tar-Minyatur) [reply Localization/de-DE/home.yaml:27]: I like it. Also helps to beef up the line a bit...looks a bit lost with its three letter. ;)

Comment (max-dau) [reply Localization/de-DE/home.yaml:27]: so true. :)

File: Localization/de-DE/home.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+#"Find the version of the software you prefer and get started. Downloading Folding@home is completely free, easy to install and safe to use.#
+#
+#Available for:
+#
+#Linux
+#
+#Windows
+#
+#Mac",
+"Suchen Sie sich die Version der Software aus, die ihnen am liebsten ist und legen Sie los. Folding@home herunterzuladen ist komplett kostenlos, es ist einfach zu installieren und die Nutzung ist sicher.
```
Comment (dsdel) [inline Localization/de-DE/home.yaml:19]:
```suggestion
"Suchen Sie sich die Version der Software aus, welche Ihnen am liebsten ist und legen Sie los. Folding@home ist komplett kostenlos, einfach zu installieren und die Nutzung ist sicher.
```
- Changed _die_ zu _welche_
- Uppercase _Ihnen_ 
- Removed some words - imho easier to read and more /marketing/ potential

**Pull request merged**